defaults/main.yml

```diff
@@ -37,4 +37,5 @@ uca_apt_repo_url: "http://ubuntu-cloud.archive.canonical.com/ubuntu"
 
 keepalived_instances: []
 keepalived_sync_groups: {}
-keepalived_bind_on_non_local: False
\ No newline at end of file
+keepalived_bind_on_non_local: False
+keepalived_options: "-D"
```

Comment:
I'd rather have this defaulted to nothing. This way we don't change standard behavior.